[bug][build] Could not resolve dependencies  org.lz4:lz4-java:jar:1.7.1

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.1.1:shade (shade-flink) on project flink-connector-mysql-cdc: Error creating shaded jar: Could not resolve following dependencies: [org.lz4:lz4-java:jar:1.7.1 (test)]: Could not resolve dependencies for project com.ververica:flink-connector-mysql-cdc:jar:2.1-SNAPSHOT: Could not transfer artifact org.lz4:lz4-java:jar:1.7.1 from/to central (https://repo.maven.apache.org/maven2): transfer failed for https://repo.maven.apache.org/maven2/org/lz4/lz4-java/1.7.1/lz4-java-1.7.1.jar: Connection timed out (Read failed) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :flink-connector-mysql-cdc


https://dev.azure.com/ververica/flink-cdc-connectors/_build/results?buildId=417&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=ce18b30f-11b8-5f1e-8fbd-391708956df7

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -30,7 +30,7 @@ steps:
 - task: Maven@3
   inputs:
     mavenPomFile: 'pom.xml'
-    mavenOptions: '-Xmx3072m'
+    mavenOptions: '-Xmx3072m -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120'
     javaHomeOption: 'JDKVersion'
     jdkVersionOption: '1.8'
     jdkArchitectureOption: 'x64'

